_includes/components/header--extended.html

```diff
@@ -1,85 +0,0 @@
-{% if header %}
```

Comment:
This file is no longer needed as the uswds-jekyll version of this file has been updated